bin/bunyan

```diff
@@ -849,12 +857,16 @@ function emitRecord(rec, line, opts, stylize) {
         }
         delete rec.req_id;
 
-        var onelineMsg;
-        if (rec.msg.indexOf('\n') !== -1) {
-            onelineMsg = '';
-            details.push(indent(stylize(rec.msg, 'cyan')));
-        } else {
-            onelineMsg = ' ' + stylize(rec.msg, 'cyan');
+        var onelineMsg = '';
+        if (!tiny) {
+            if (rec.msg.indexOf('\n') !== -1) {
```

Comment:
Why not do this as:

```
if (tiny) {
  ...
} else if (rec.msg.indexOf('\n') !== -1) {
```

rather than indenting all of the existing `if`:s one extra step?